smi/internal/module.go

```diff
@@ -304,6 +305,14 @@ func GetModulePath(name string) (string, error) {
 	return "", os.ErrNotExist
 }
 
+func GetModuleFromStream(name string, data []byte) (*Module, error) {
+	module := FindModuleByName(name)
+	if module != nil {
+		return module, nil
```

Comment:
should it be considered an error to load a new module by the same name of an existing module?  It seems that otherwise `data` is unused.

Reply:
When loading mib files in a batch, it's probably common to have same modules loaded multiple times.  It's the parsing errors that we are mostly concerned of.  so the next line returns results from LoadModuleFromStream()

Reply:
But isn't this func handling a single mib? Seems that the caller should be able to decide if their batch of mibs should contain dups when calling this func one mib at a time.

Reply:
consider this use case: a user load all mib files for router A, and later, decide to do the same for router B.
He/she will go to the vendor's website and download all mib files.  Depending on the platform, it's possible that the two sets of mib files have duplicates.
In this case, it probably will be time consuming for the user to de-duplicate the files.
btw, this function is based on `GetModule()`